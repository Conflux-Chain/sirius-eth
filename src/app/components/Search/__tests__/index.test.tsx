import React from 'react';
import { render } from '@testing-library/react';
import { Search } from '..';
describe('<Search  />', () => {
  it('should match snapshot', () => {
    const SearchComp = render(<Search>Search</Search>);
    expect(SearchComp).toMatchSnapshot();
  });
});
