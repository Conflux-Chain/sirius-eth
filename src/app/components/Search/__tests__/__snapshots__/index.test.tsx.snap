// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Search  /> should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .c0 input::-webkit-input-placeholder {
  white-space: nowrap;
}

.c0 input::-moz-placeholder {
  white-space: nowrap;
}

.c0 input:-ms-input-placeholder {
  white-space: nowrap;
}

.c0 input::placeholder {
  white-space: nowrap;
}

<body>
    <div>
      <div
        class="undefined"
      >
        <div
          class="jsx-1159070929 with-label"
        >
          <div
            class="jsx-1159070929 input-container c0 undefined"
          >
            <div
              class="jsx-1159070929 input-wrapper line    "
            >
              <input
                autocomplete="off"
                class="jsx-1159070929 right-icon"
                placeholder=""
                type="text"
                value=""
              />
              <span
                class="jsx-2293214799 input-icon"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="undefined"
    >
      <div
        class="jsx-1159070929 with-label"
      >
        <div
          class="jsx-1159070929 input-container sc-AxjAm nYPfk undefined"
        >
          <div
            class="jsx-1159070929 input-wrapper line    "
          >
            <input
              autocomplete="off"
              class="jsx-1159070929 right-icon"
              placeholder=""
              type="text"
              value=""
            />
            <span
              class="jsx-2293214799 input-icon"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
